packages/server/src/arpa_reporter/routes/audit-report.js

```diff
@@ -105,6 +105,25 @@ router.get('/', requireUser, async (req, res) => {
     res.send(Buffer.from(report.outputWorkBook, 'binary'));
 });
 
+router.post('/refresh-cache', async (req, res) => {
```

Comment:
I brought this up on a prior call. I am not entirely sure. Without production access, I'm relying on USDR to let me know their best workflow.

I assumed they had some way to call authenticated endpoints locally through a tool like postman, in which case this would just be another endpoint in postman that has some auth on it.

It's something we can discuss on the next call.